src/WebUI/Pods/PodsRightPanel.tsx

```diff
@@ -62,6 +63,10 @@ export class PodsRightPanel extends BaseComponent<IPodRightPanelProps, IPodsRigh
         );
     }
 
+    private _getHeader(): JSX.Element | null {
+        return (<h2 className="pod-right-panel-header">{(this.props.pod.metadata && this.props.pod.metadata.name) || ""}</h2>);
```

Comment:
we should check pipeline code, whether they have used h2 or not?

Reply:
Added a Todo to move all headers appropriately